proposals/Readme.md

```diff
@@ -5,3 +5,10 @@ Create new proposals here, in their own directories.
 A Readme.md file should exist in each directory, and may contain 
 the entire proposal, or it may serve as an index to the rest of the materials
 within the proposal.
+
+## Current Proposals
```

Comment:
I arbitrarily opted to have list in alpha-order. 

The status badges can be tweaked, or we can ditch them altogether.

Reply:
I like the badges